### ISSUE TITLE
Fix the website

### ISSUE THREAD

**Issue by wimjongman** [eclipse-mylyn/mylyn-website#8]
A general issue to update the website

**Comment by BeckerFrank** [comment]
Yes we now can do all  the fix. The technical problems are solved see https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3513
